examples/scenes/src/test_scenes.rs

```diff
@@ -683,10 +699,12 @@ fn gradient_extend(scene: &mut Scene, params: &mut SceneParams) {
     }
     let extend_modes = [Extend::Pad, Extend::Repeat, Extend::Reflect];
     for (x, extend) in extend_modes.iter().enumerate() {
-        for y in 0..2 {
-            let is_radial = y & 1 != 0;
+        for (y, kind) in [Kind::Linear, Kind::Radial, Kind::Sweep]
+            .into_iter()
+            .enumerate()
```

Comment:
These now go off-screen in the default resolution. I think the `resolution` field of `params` can be set to counteract this (see #453)

Reply:
I wasn't aware this existed! Added to this scene and made sure it scales properly.